lms/djangoapps/ccx/utils.py

```diff
@@ -128,6 +128,19 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_ccx_for_coach_by_ccx_id(course, coach, ccx_id):
```

Comment:
yes i want to verify that 
The ccx I just open is the same ccx of same user on same master course .